Makefile.toml

```diff
@@ -114,12 +114,7 @@ dependencies = [
 description = "Run all tests for the CI 'wasm' job"
 category = "CI"
 dependencies = [
-    # note: CI does not call `cargo make ci-job-wasm` since
-    # we have to set up the environment for the emscripten job separately
-    # Instead, each of these is called individually.
-    "wasm-release",
-    "wasm-test-release",
-    "wasm-cpp-emscripten",
+    "test-cpp-emscripten",
```

Comment:
I think with this change the regular `ava` wasm tests aren't being run anymore

We also should probably be running the webpack build here just in case.

Reply:
`npm build` is not building any Rust. `npm install` is, and `npm publish` will build it and publish the generated files. Clients will not need to compile Rust.

I've moved the npm package into `ffi/diplomat/js/examples`, because it's an example use of the JS bindings. We might publish it in the future, but at the moment it's an example that uses a specific combination of features, Rust compiler flags, and WASM optimization. This was also the reason I gave it a build script instead of relying on some combination of `cargo make` commands to run scripts distributed over two other files.

Reply:
Yes, but the _tests_ aren't being run anymote

Reply:
They are run in the ffi job under the `test-js` task.